src/core/main.cpp

```diff
@@ -120,36 +120,36 @@ void printHelp()
 {
 	printf( "LMMS %s\n"
 		"Copyright (c) 2004-%s LMMS developers.\n\n"
-		"usage: lmms [ -r <project file> ] [ options ]\n"
+		"Usage: lmms [ -r <project file> ] [ options ]\n"
```

Comment:
In my Debian 8 system, passing `--help` to commands like `rm`, `cp`, `ls`, and `mkdir` outputs the word `Usage` in uppercase.